src/python/opentx.py

```diff
@@ -16,8 +16,8 @@ def gen_multi_bin(source, target, env):
 
 
 def gen_elrs(source, target, env):
-    if not "_stock" in env['PIOENV']:
-        return
+    #if not "_stock" in env['PIOENV']:
```

Comment:
Not sure what this check was doing before, but did you mean to comment it out? If you did, is there any reason to keep the commented code rather than just removing it completely?

Reply:
Commented it now and will be removed later when R9M targets are merged.
Now `firmware.elrs` is genered for stlink target as well. Bootloader may flash frk and elrs files nowadays.

Reply:
Cool, seems fine to me then.